mne/simulation/_metrics.py

```diff
@@ -32,19 +30,15 @@ def source_estimate_quantification(stc1, stc2, metric='rms'):
         Second source estimate for comparison.
     metric : str
         Metric to calculate, 'rms' or 'cosine'.
-
     Returns
     -------
     score : float | array
         Calculated metric.
-
```

Comment:
These removals are going to break the rendering. Perhaps this was copy-paste from GitHub in a browser? Sometimes it does not properly copy the newlines...